src/lib/indexer/__tests__/eventProcessor.test.ts

```diff
@@ -26,36 +26,41 @@ describe('eventProcessor', function () {
       )
 
       expect(result).to.deep.equal({
-        examples: new Map([
+        certificates: new Map([
           ['42', { type: 'update', id: '42', state: 'created', latest_token_id: 1, original_token_id: 1 }],
         ]),
       })
     })
 
-    it("should return new attachment and example if transaction doesn't exist", function () {
-      const result = eventProcessors['example-create'](
+    it("should return new certificate if transaction doesn't exist", function () {
+      const result = eventProcessors['process_initiate_cert'](
         1,
         null,
         'alice',
         [],
         [
           {
             id: 1,
-            roles: new Map([['owner', '123']]),
+            roles: new Map([['owner', 'emma-hydrogen-producer']]),
```

Comment:
Emma is the energy producer, Heidi is the hydrogen producer

Reply:
thanks for a highlight:)